File: helfomat-api/src/main/java/de/helfenkannjeder/helfomat/api/organisation/event/OrganizationEditNameEventDto.java
package de.helfenkannjeder.helfomat.api.organisation.event;

import de.helfenkannjeder.helfomat.core.organisation.OrganisationId;

/**
 * @author Valentin Zickner
 */
public class OrganizationEditNameEventDto extends OrganizationEventDto {
    private String name;

    OrganizationEditNameEventDto() {
    }

    public OrganizationEditNameEventDto(OrganisationId organisationId, String name) {
        super(organisationId);
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: helfomat-api/src/main/kotlin/de/helfenkannjeder/helfomat/api/organization/event/OrganizationEditNameEventDto.kt
package de.helfenkannjeder.helfomat.api.organization.event;

import de.helfenkannjeder.helfomat.core.organization.OrganizationId;

/**
 * @author Valentin Zickner
 */
public class OrganizationEditNameEventDto extends OrganizationEventDto {
    private String name;

    OrganizationEditNameEventDto() {
    }

    public OrganizationEditNameEventDto(OrganizationId organizationId, String name) {
        super(organizationId);
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public <T> T visit(OrganizationEventDtoVisitor<T> visitor) {
        return visitor.visit(this);
    }

}
